Pokedex/Pokedex/CustomCells/CustomCollectionViewCell.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CustomCollectionViewCell.swift
+//  Pokedex
+//
+//  Created by Cameron Dunn on 1/25/19.
+//  Copyright © 2019 Cameron Dunn. All rights reserved.
+//
+
+import UIKit
+
+class CustomCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var pokeImage : UIImageView!
+}
```

Comment:
Then check the pokemon and create and image from it's url
```suggestion
var pokemon: Pokemon?
    {
        didSet { updateViews() }
    }
    
    var pokemonImage: UIImage!
       func updateViews() {
            guard let pokemon = pokemon,
            let imageURLstring = pokemon.imageLink.frontDefault,
            let imageURL = URL(string: imageURLstring) else { return }
        do {
            let data = try Data(contentsOf: imageURL)
            pokemonImage = UIImage(data: data)
        } catch {
            NSLog("Error converting url to image: \(error)")
        }
        
        DispatchQueue.main.async {
            guard let pokemonImage = self.pokemonImage else { return }
            self.pokeImage.image = pokemonImage
        }
    }
}
```